srtcore/core.cpp

```diff
@@ -8477,7 +8478,7 @@ int CUDT::processData(CUnit *in_unit)
                 continue;
             }
 
-            const int avail_bufsize = m_pRcvBuffer->getAvailBufSize();
+            avail_bufsize = m_pRcvBuffer->getAvailBufSize();
```

Comment:
Restore `const int` (discard this change).

Reply:
No. This is how it should be.

Reply:
Well, looks like I was wrong about the necessity of extending the scope for this variable. Things have changed dramatically after introducing a loop for the needs of packet filter.